infra/equaler.go

```diff
@@ -1,4 +1,4 @@
-package models
+package infra
```

Comment:
Yeah, Agreed. "infra" is very generic name.  As of now it contains Equaler, DummyEqualer and Lifecycle interfaces. So it is not gorm only specific. Will try to rename more specifics (maybe need to split into 2).